add updated tar links (#67665)
openshift/openshift-docs: modules/op-release-notes-1-11.adoc

* With this update, you can use the skopeo tool with {tekton-chains} to generate keys, which are used in the `cosign` signing scheme.

> * When you upgrade to the {pipelines-title} Operator 1.11, the previous {tekton-chains} configuration will be overwritten and you must set it again in the `TektonConfig` CR.

Reviewer: ```suggestion
* When you upgrade to the {pipelines-title} Operator 1.11, the previous {tekton-chains} configuration is overwritten and you must set it again in the `TektonConfig` CR.
```

Reviewer: https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc 

> Use active voice and present tense.

Author: Hello @xenolinux, thank you for your review. I'd like to clarify that both sentences mentioned are written in active voice. You might have concerns regarding the use of future tense, as advised against in the IBM style guide, but I think the se of future tense is intentional here. The sentence with "will" suggests that the configuration's overwriting occurs in the future, right after completing the upgrade. In contrast, the sentence with "is" uses present tense, indicating that the overwriting is an immediate result of the upgrade, happening almost at the same time. This is a follow up PR, to a [PR](https://redhat-internal.slack.com/archives/D05UYR89RSQ/p1699543917093769) that was previously reviewed by peer review squad. In that PR stylistic changes were applied.
